appctl_msg_defs/package.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<package>
+  <name>appctl_msg_defs</name>
+  <version>0.0.1</version>
+  <description>Support software for mode switching</description>
+
+  <maintainer email="lg@endpoint.com">End Point Corp</maintainer>
+
+  <author email="wojtek@endpoint.com">Wojciech Ziniewicz</author>
+
+  <url type="website">https://liquidgalaxy.endpoint.com/</url>
+
+  <license>Apache 2.0</license>
+
+  <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>rospy</build_depend>
```

Comment:
This package probably just needs catkin and message gen/runtime.  Refer to [the std_msgs package](https://github.com/ros/std_msgs/blob/kinetic-devel/package.xml) which also defines messages with no msg dependencies.  Same for CMakeLists.txt

Reply:
Cool, thanks for the example.  Makes sense.